website/docs/reference/api-tokens-and-client-keys.mdx

```diff
@@ -55,6 +56,18 @@ Do **not** use admin tokens for:
 
 Support for scoped admin tokens with more fine-grained permissions is currently in the planning stage.
 
+
+### Project level access tokens
+
+All project members can see any `CLIENT` (server-side client token) and `FRONTEND` (client-side client token) respectively.
+
+Use personal access tokens to:
+
+- Provide more fine-grained permissions for automation than an admin token provides
+- Give access to all project members (or anyone with `READ_PROJECT_API_TOKEN` permission) to read the token
+- Give access to the project owner (or anyone with `CREATE_PROJECT_API_TOKEN` permission) to create a token,
```

Comment:
If you agree with my suggestion regarding visibility, then I think we can also delete this and the rest of the other content from this section. 

But before you do: isn't the `read_project_api_token` permission a project-level permission? So don't you have to be BOTH a project member AND have that permission? And the same for creating.

Reply:
Yes, we assign the `read_project_api_token` permission to Project Members by default and the same for project owner and `create_project_api_token`